components/Feature/Services/Categories/index.js

```diff
@@ -1,30 +1,52 @@
-import CategoryCard from './CategoryCard';
-import styles from './index.module.scss';
+const Categories = ({ categorisedResources, selectedCategories, setSelectedCategories }) => {
+  const categoryProblems = {
+    'Loneliness or isolation': { problemDescription: 'Loneliness or isolation' },
```

Comment:
Could probably not add this line and `Food or shopping` cause they're the same as category names

Reply:
Sorted!